### PR TITLE
Facedetection application files

### PR DESCRIPTION
**Pull request by oguzhanbsolak**

### CODE REVIEW

File: scripts/gen_facedet_tinierssd_max78002.sh
Comment (MaximGorkem) [inline scripts/gen_facedet_tinierssd_max78002.sh:1]: Maybe we could add max78002 or AI87 to the prefix name to differentiate it from the AI85 folder.. It won't produce the folder if the "scripts/gen_facedet_tinierssd_max78000.sh" is already run.

**Review by MaximGorkem — APPROVED** [review]
Looks good. I have just a small comment about the ai8xize prefix naming.

File: networks/ai85-facedet-tinierssd.yaml [hunk 1]
```diff
@@ -0,0 +1,191 @@
+---
+# HWC (little data) configuration for FaceDetection
+# Parallel Model
```
Comment (ermanok) [inline networks/ai85-facedet-tinierssd.yaml:3]: What does 'parallel model' stands for?

**Review by ermanok — APPROVED** [review]
Looks good to me. I just cannot understand a comment in the YAML file, please have look if that line requires more explanations...

File: scripts/gen_facedet_tinierssd_max78002.sh
Comment (oguzhanbsolak) [reply scripts/gen_facedet_tinierssd_max78002.sh:1]: The spaces were not visible in my previous comment

<pre>
-sdk
  -Examples
    -MAX78000
       -CNN
         -facedet_tinierssd
    -MAX78002
        -CNN
          -facedet_tinierssd
</pre>

File: networks/ai85-facedet-tinierssd.yaml (hunk 1 above)
Comment (oguzhanbsolak) [reply networks/ai85-facedet-tinierssd.yaml:3]: Thank you for the comment. The 'Parallel Model' term comes from the original tinierssd model, and I believe it stands for the parallel outputs from different layers. I think we may remove it as it is not critical information.

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**